### PR TITLE
Add IT in pipeline

### PR DESCRIPTION
**Pull request by rishabh-11**
**What this PR does / why we need it**:
Adds IT in the pipeline. 
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement developer
Addition of the existing integration tests in the pipeline. IT will now be run on pipeline for every PR
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,20 +19,20 @@ IMAGE_TAG           := $(shell cat VERSION)
...
 MC_IMAGE			:=
-# MCM_IMAGE			:= eu.gcr.io/gardener-project/gardener/machine-controller-manager:v0.39.0
-# MC_IMAGE			:= $(IMAGE_REPOSITORY):v0.7.0
+# MCM_IMAGE			:= eu.gcr.io/gardener-project/gardener/machine-controller-manager:v0.46.0
+# MC_IMAGE			:= $(IMAGE_REPOSITORY):v0.6.0
 LEADER_ELECT 	    := "true"
 # If Integration Test Suite is to be run locally against clusters then export the below variable
 # with MCM deployment name in the cluster
-MACHINE_CONTROLLER_MANAGER_DEPLOYMENT_NAME := machine-controller-manager
+#MACHINE_CONTROLLER_MANAGER_DEPLOYMENT_NAME := machine-controller-manager
```
Comment (himanshu-kun) [inline Makefile:35]:
```suggestion
# MACHINE_CONTROLLER_MANAGER_DEPLOYMENT_NAME := machine-controller-manager
```

File: .ci/pipeline_integration_test [hunk 2]
```diff
@@ -0,0 +1,195 @@
+############################################## <Color> ##############################################
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+############################################## </Color> ##############################################
+
+mkdir -p dev
+
+logs_path=.ci/controllers-test/logs
```
Comment (himanshu-kun) [inline .ci/pipeline_integration_test:8]: make a logs folder at this path like done for other providers

**Review by himanshu-kun — COMMENTED** [review]
first review done

File: .ci/pipeline_integration_test (hunk 2 above)
Comment (rishabh-11) [reply .ci/pipeline_integration_test:8]: we already have a logs folder

Comment (himanshu-kun) [reply .ci/pipeline_integration_test:8]: okay

**Review by shaoyongfeng — APPROVED** [review]

**Review by himanshu-kun — APPROVED** [review]
/lgtm

**Comment by himanshu-kun** [comment]
one more tick for the issue https://github.com/gardener/machine-controller-manager/issues/631 🥳

**Pull request merged**